src/slacker/client/cluster.clj

```diff
@@ -211,8 +211,7 @@
           (parse-grouping-options call-options
                                   ns-name func-name params)
           slacker-ns-servers (get-ns-mappings this)
-          target-servers (find-server this slacker-ns-servers ns-name
-                                      (partial grouping* ns-name func-name params))
+          target-servers (find-server this slacker-ns-servers ns-name grouping*)
```

Comment:
`ns-name`, `func-name` and `params` is already added to `grouping*` in `parse-grouping-options` on line 221